src/config/symbol.rs

```diff
@@ -545,7 +545,7 @@ impl Symbol {
 
 impl Display for Symbol {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{} kind:{} addr:{:#x}", self.name, self.kind, self.addr)?;
+        write!(f, "{} kind:{} addr:0x{:08x}", self.name, self.kind, self.addr)?;
```

Comment:
```suggestion
        write!(f, "{} kind:{} addr:{:#08x}", self.name, self.kind, self.addr)?;
```
Very minor, but # will emit the 0x prefix.

Reply:
Thanks, that's good to know. `0x{:08x}` is how it's written everywhere else in this project, so I copied it for consistency.

Reply:
That looks better in my opinion, I'll update the codebase to use `{:#08x}` in a separate commit.